### PR TITLE
test: Events API integration test - Success Case

### PR DESCRIPTION
**Pull request by nimat1**
Added 'Events' API integration test.
Added one CMS file and one Data file 
Passed test screen shot 
image

### CODE REVIEW

File: integration-test/tests/cms/events.test.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {expect, test} from '@playwright/test';
+import { eventsExepctedInformation } from '../../utils/datafactory/events.data';
+test('GET /api/cms/v1/code-of-conduct returns correct team data', async ({request}) => {
+    const response = await request.get(`https://wcc-backend.fly.dev/api/cms/v1/events`);
+ 
+      if (response.status() === 200) {
```
Comment (dricazenck) [inline integration-test/tests/cms/events.test.ts:6]: Could you please use assertions for that

File: integration-test/tests/cms/codeofconduct.test.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@playwright/test';
+import { codeofconductExepctedInformation } from '../../utils/datafactory/codeofconduct.data';
+test('GET /api/cms/v1/code-of-conduct returns correct team data', async ({request}) => {
+    const response = await request.get(`https://wcc-backend.fly.dev/api/cms/v1/code-of-conduct`);
+ 
+      if (response.status() === 200) {
```
Comment (dricazenck) [inline integration-test/tests/cms/codeofconduct.test.ts:6]: Please use assertions, check existent tests done by me and Nora.

File: integration-test/tests/cms/codeofconduct.test.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@playwright/test';
+import { codeofconductExepctedInformation } from '../../utils/datafactory/codeofconduct.data';
+test('GET /api/cms/v1/code-of-conduct returns correct team data', async ({request}) => {
+    const response = await request.get(`https://wcc-backend.fly.dev/api/cms/v1/code-of-conduct`);
```
Comment (dricazenck) [inline integration-test/tests/cms/codeofconduct.test.ts:4]: You can use only path, the domain is already defined by env variables

File: integration-test/tests/cms/events.test.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import {expect, test} from '@playwright/test';
+import { eventsExepctedInformation } from '../../utils/datafactory/events.data';
+test('GET /api/cms/v1/events returns correct data', async ({request}) => {
+    const response = await request.get(`/api/cms/v1/events`);
+ 
+    expect(response.status()).toBe(200);
+   
+    const body = await response.json();
+  
+    const expectedResponse = eventsExepctedInformation;
```
Comment (dricazenck) [inline integration-test/tests/cms/events.test.ts:10]: for now you can just check for status code, and remove the content check and the json.

**Comment by nimat1** [comment]
@dricazenck - Done the suggested changes. Thanks !

File: integration-test/tests/cms/events.test.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import {expect, test} from '@playwright/test';
+test('GET /api/cms/v1/events returns correct data', async ({request}) => {
+    const response = await request.get(`/api/cms/v1/events`);
+    // response status validation
+    expect(response.status()).toBe(200);
```
Comment (dricazenck) [inline integration-test/tests/cms/events.test.ts:5]: Please could you fix the format

File: integration-test/tests/cms/events.test.ts [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import {expect, test} from '@playwright/test';
+test('GET /api/cms/v1/events returns correct data', async ({request}) => {
```
Comment (dricazenck) [inline integration-test/tests/cms/events.test.ts:2]:
```suggestion
test('GET /api/cms/v1/events returns success response code', async ({request}) => {
```

File: integration-test/utils/datafactory/codeofconduct.data.ts [hunk 7]
```diff
@@ -0,0 +1,38 @@
+export const codeofconductExepctedInformation = {
```
Comment (dricazenck) [inline integration-test/utils/datafactory/codeofconduct.data.ts:1]: this data is not related to this PR, could you please remove from this PR.

File: integration-test/tests/cms/events.test.ts [hunk 8]
```diff
@@ -0,0 +1,5 @@
+import {expect, test} from '@playwright/test';
+test('GET /api/cms/v1/events returns success response code', async ({request}) => {
+    const response = await request.get(`/api/cms/v1/events`);
+    expect(response.status()).toBe(200);
+     });
```
Comment (dricazenck) [inline integration-test/tests/cms/events.test.ts:5]:
```suggestion
});
```

File: integration-test/tests/cms/events.test.ts [hunk 9]
```diff
@@ -0,0 +1,5 @@
+import {expect, test} from '@playwright/test';
+test('GET /api/cms/v1/events returns success response code', async ({request}) => {
```
Comment (dricazenck) [inline integration-test/tests/cms/events.test.ts:2]:
```suggestion

test('GET /api/cms/v1/events returns success response code', async ({request}) => {
```

**Review by dricazenck — APPROVED** [review]
